### ISSUE TITLE
Tests: UI Test

### ISSUE THREAD

**Issue by levey** [levey/AwesomeMenu#62]
Things I'd like if you're adding tests for this project

**Tooling**: KIF

**Areas to look at**:

- Expecting presence of startPoint menu item element after setup menu.
- Expecting presence of each option menu item element after expanding menu.
- Expecting absence of each option menu item element after closing menu.

**Note**
In anticipation of [Test Jam](http://blog.cocoapods.org/Test-Jammin/)

**Comment by AmitAjmera** [comment]
I have integrated Awesome menu code in my App now problem is that this menu shows in iOS simulator but not showing on iOS devices. Am I forgetting to configure something or anything else that I am missing?

**Comment by world** [comment]
We have been struggling mightily to get XCUI to be able to query awesomemenu's menu buttons, such that it can understand which item was actually selected after the menu animates. But something about awesomemenuitems being housed inside awesomemenu's view (which in turn is embedded in whatever xib holds actually contains awesomemenu is confusing XCUI such that we just can't seem to test awesomemenu. We really do t want to rip it out but without test coverage we might have to. Any ideas? :)

**Comment by theonetheycallneo** [comment]
Yes, being able to access or even customize the accessibilityLabels would be ideal for the AwesomeMenuItems +1
